docs/ignoring-files-folders-code.md

```diff
@@ -12,10 +12,12 @@ import IgnoreIndividualFindings from "/src/components/procedure/_ignore-individu
 
 Exclude specific files, folders or parts of code from results of Semgrep scans in your repository or working directory. Semgrep does not generate findings for the ignored items.
 
-This is separate from [ignoring a **finding**](../managing-findings), which is a triage operation after a Semgrep scan has found a match.
-
 All Semgrep environments (CLI, CI, and App) adhere to user-defined or Semgrep-defined ignore patterns.
 
+:::info
+Ignoring files or folders is different from ignoring a **finding**, which is a triage operation after a Semgrep scan has found and reported a match. See [Finding status](/managing-findings/#finding-status) for more information about ignored findings. Ignored files, and folders are skipped, ignored findings were reported, and then manually ignored by users.
```

Comment:
```suggestion
Ignoring files or folders is different from ignoring a **finding**, which is a triage operation after a Semgrep scan has found and reported a match. To triage findings as "ignored", see [Ignoring findings](semgrep-code/findings/#ignoring-findings). This document focuses on ignoring files and folders, which means these are skipped by Semgrep and **not** scanned.
```

Reply:
Used parts of this suggestion above in https://github.com/returntocorp/semgrep-docs/pull/986#discussion_r1183969301